### DISCUSSION TITLE
Override source in ApplicationSet, via file on git.

### DISCUSSION THREAD

**Discussion by QF0xB** [argoproj/argo-cd#14281] [Q&A] [answered]
Hey!

I am trying to setup my CD pipeline.

For this I would need to have a placeholder in the ApplicationSet for targetRevision. This placeholder should be replace by a string in a file on a different git repo. Is that possible?

Best regards.

**Comment by crenshaw-dev** [comment, marked as answer]
You could potentially use a git files generator to pull revision info from another repo.

Or you could use [git-stored source overrides](https://argo-cd.readthedocs.io/en/latest/user-guide/parameters/#store-overrides-in-git) if it's okay for the file to be located in the source repo.

**Reply by QF0xB** [reply]
Yes that did work. Thank you a lot
